scripts/build_upload.sh

```diff
@@ -96,7 +96,7 @@ echo "Removing first bokeh build at $first_build_loc"
 #########################
 
 # build for each python version
-for py in 27 33 34;
```

Comment:
well... we can still provide the py33 packages... but if we are not testing in py33, we could be shipping a non working thing... I think we need to be consistent...